db/models/assessment_record/assessment_records.py

```diff
@@ -57,6 +58,8 @@ class AssessmentRecord(db.Model):
         Computed(func.md5(cast(jsonb_blob, TEXT)), persisted=True),
     )
 
+    scores = relationship("JustScore")
```

Comment:
⛏️ Make sure that this change is reflected in the queries for `assessment_records`, I can assist with this :)

Reply:
Which queries would need updating in assessment records?